cardano-api/src/Cardano/API.hs

```diff
@@ -191,6 +191,7 @@ module Cardano.API (
     deserialiseFromTextEnvelope,
     readFileTextEnvelope,
     writeFileTextEnvelope,
+    writeTextEnvelopeFileWithOwnerPermissions,
```

Comment:
Lets follow the name scheme we have with `writeFileTextEnvelope`, so it'd be `writeFileTextEnvelopeWithOwnerPermissions`.